src/main/controlMod.F90

```diff
@@ -371,6 +375,11 @@ subroutine control_init( )
                errMsg(sourcefile, __LINE__))
        end if
 
+       if (itmax_hybrid < 1) then
+          call endrun(msg=' ERROR: expecting itmax_hybrid > 0 where 40 is the default value ' // &
```

Comment:
I'd remove the statement of default value here - otherwise this will surely get out of sync with the actual default over time.